Jenkinsfile

```
@@ -224,7 +224,8 @@ String functional_packages(String distro) {
             "hdf5-vol-daos-mpich2-tests-daos-0 " +
             "hdf5-vol-daos-openmpi3-tests-daos-0 " +
             "MACSio-mpich2-daos-0 " +
-            "MACSio-openmpi3-daos-0"
+            "MACSio-openmpi3-daos-0 " +
+            "mpifileutils-0"
```

I don't think this is going to work.  There is nothing in https://github.com/daos-stack/mpifileutils-pkg/pull/2/files#diff-64772e141dec1b846975136106159765bf76121ac4312a7eeaa0e1d84e009ca8 that is going to _provide_ **mpifileutils-0**.

I suspect you want:
```suggestion
            "mpifileutils-daos-mpich"
```

Although I do generally agree that it should be something like:

```suggestion
            "mpifileutils-mpich-daos-0"
```

However daos-stack/mpifileutils-pkg#2 doesn't have anything providing that.  That said, I am having second thoughts on all of this need for _daos-0_ virtual provides.  I'd need to do some experimenting to be sure we can eliminate it though.

If you use daos-stack/mpifileutils-pkg#1 you should be able to use:

```suggestion
            "mpifileutils-mpich-daos-0"
```

---

@saurabhtandan Will you change your commit message to:
`PR-repos-el7: mpifileutils-pkg@PR-1`
And change `mpifileutils-0` to `mpifileutils-mpich-daos-0`
And then repush please? Sorry about that!

---

> @saurabhtandan Will you change your commit message to:
> `PR-repos-el7: mpifileutils-pkg@PR-1`
> And change `mpifileutils-0` to `mpifileutils-mpich-daos-0`
> And then repush please? Sorry about that!

Done.